src/pages/ReportDetailsPage.tsx

```diff
@@ -217,6 +223,13 @@ function ReportDetailsPage({policies, report, session, personalDetails}: ReportD
         Report.leaveGroupChat(report.reportID);
     }, [isChatRoom, isPolicyEmployee, isPolicyExpenseChat, report.reportID, report.visibility]);
 
+    const unapproveExpenseReportOrShowModal = useCallback(() => {
+        // TODO: show modal if report is exported to accounting
```

Comment:
will do this whenever it's ready - https://expensify.slack.com/archives/C06ML6X0W9L/p1718955793112739?thread_ts=1718663737.743339&cid=C06ML6X0W9L